src/views/Submit.vue

```diff
@@ -145,6 +145,21 @@ export default {
 
 	mixins: [ViewsMixin],
 
+	/*
+     * This is used to confirm that the user wants to leave the page
+     * if the form is unsubmitted.
+     */
```

Comment:
```suggestion
	 * This is used to confirm that the user wants to leave the page
	 * if the form is unsubmitted.
	 */
```

Mix of tab and spaces

Reply:
Oh, some copy paste leftovers... 🙈